### PR TITLE
fix(zk_toolbox): readme added dependencies section and cleaned up

### PR DESCRIPTION
**Pull request by zk-Lumi**
## What ❔
- Added dependencies section
- Added `bash` to code blocks
- Made all major commands code blocks for easier copying and better styling
- Added fixed version to rust toolchain (1.78.0)

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] Code has been formatted via `zk fmt` and `zk lint`.
- [x] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

File: zk_toolbox/README.md [hunk 1]
```diff
@@ -7,15 +7,29 @@ Toolkit for creating and managing ZK Stack chains.
...
+
+Ensure you have followed
+[these instructions](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/setup-dev.md) to set up
+dependencies on your machine (don't worry about the Environment section for now).
+
+In addition to the above instructions, you will also need to
+[install Foundry](https://book.getfoundry.sh/getting-started/installation).
+
+IMPORTANT: Ensure you are running the latest stable releases of Rust using `rustup update stable` (rustc 1.78.0) & cargo
+using `cargo update` (cargo 1.78.0).
```
Comment (Deniallugo) [inline zk_toolbox/README.md:20]: just place necessary version `1.78.0` to https://github.com/matter-labs/zksync-era/blob/main/zk_toolbox/rust-toolchain

File: zk_toolbox/README.md [hunk 2]
```diff
@@ -7,15 +7,29 @@ Toolkit for creating and managing ZK Stack chains.
...
 all necessary arguments via the command line.
 
+### Dependencies
+
+Ensure you have followed
+[these instructions](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/setup-dev.md) to set up
+dependencies on your machine (don't worry about the Environment section for now).
+
+In addition to the above instructions, you will also need to
+[install Foundry](https://book.getfoundry.sh/getting-started/installation).
```
Comment (Deniallugo) [inline zk_toolbox/README.md:17]: should we add it to the main instruction?

File: zk_toolbox/README.md (hunk 1 above)
Comment (zk-Lumi) [reply zk_toolbox/README.md]: Done 👌 Can you check zk_toolbox/rust-toolchain commit for format? I think its `stable-1.78.0' but not positive it's not just `1.78.0`

**Review by Deniallugo — DISMISSED** [review]

File: zk_toolbox/README.md (hunk 2 above)
Comment (zk-Lumi) [reply zk_toolbox/README.md]: Also CC @githubdoramon

**Pull request merged**